packages/Confirmation/src/Confirmation.js

```diff
@@ -68,6 +69,10 @@ const Confirmation = props => {
     }
   }, [confirmButtonRef, defaultIsOpen]);
 
+  React.useEffect(() => {
+    popoverKey = uuid();
+  }, [isPending]);
```

Comment:
Why do we need to force the popover rerendering?

Reply:
Yeah it was needed because when the `isPending` prop is true on the button compoennt it shows spinning arrows and causes a width change. If the popover isn't rerendered (and width evaluated correctly) it causes layout flow issues - https://github.com/acl-services/paprika/pull/343#discussion_r356345840